### DISCUSSION TITLE
ChatGPT - Writer Error Message

### DISCUSSION THREAD

**Discussion by Spanisheyez** [GorvGoyl/ChatGPT-Writer#33] [Report Bugs and Issues]
Hi, 
I have recently installed the ChatGPT- Writer plugin to my gmail. However I am constantly getting the mentioned below error. I am always logged into ChatGPT. 

There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

**Comment by GorvGoyl** [comment]
Which browser do you use? Could you also please share the screen recording in order for me to debug the issue?

**Reply by Spanisheyez** [reply]
I am using google chrome. 

here is a snapshot 

![Error](https://user-images.githubusercontent.com/123445336/214235425-0a0b4ab1-7996-4ac3-9fd5-184b106ea6e8.jpg)

**Reply by GorvGoyl** [reply]
do you see any error messages in the browser dev console? https://support.awesome-table.com/hc/en-us/articles/115003787169-Using-the-Debug-Console-in-Google-Chrome-to-identify-errors

**Reply by GorvGoyl** [reply]
also are you able to login to chat.openai.com ?

**Reply by Spanisheyez** [reply]
![Error](https://user-images.githubusercontent.com/123445336/214245531-56dca950-a673-4a8a-8bf1-dc712675d58a.jpg)
I see this error in my browser dev console

**Reply by Spanisheyez** [reply]
I am able to login to chat.openai.com successfully. And it works perfectly
fine for me.

**Reply by GorvGoyl** [reply]
hmm that error is not related to chatgpt writer. Would it be okay for you to quickly jump on the call for 10 mins and share your screen so that I can find the root cause of the issue?

**Reply by Spanisheyez** [reply]
Yeah i could do it. Please send me the call link

**Reply by GorvGoyl** [reply]
https://meet.google.com/ope-azwo-erb

**Reply by GorvGoyl** [reply]
I tested it with a fresh install of the Chrome browser and faced no issues. I'm not sure why exactly the issue is happening for you but it's related to some setting you might have changed in Chrome browser. Maybe try to reinstall the extension or as a last resort reset browser settings or reinstall the browser.

**Reply by Spanisheyez** [reply]
ok let me try. i will get back to you

**Reply by Spanisheyez** [reply]
Good news: after reinstalling the extension it solved the problem.
bad news: Now the screen alignment of chatgpt-writer pop up is out. see attached snapshot. the generate response button is hiding and there is no scrollbar available to scroll it down
![Error](https://user-images.githubusercontent.com/123445336/214276900-a220e9bb-155f-4637-801d-22473b8ddfa5.jpg)

**Reply by GorvGoyl** [reply]
I'll look into it

**Comment by Spanisheyez** [comment]
Hi,
Any updates on this?
